.gitignore

```diff
@@ -1,6 +1,7 @@
 cover.out
 .idea/
 tmp/
+ddl/
```

Comment:
ddl?

Reply:
I dumped the schema there locally so that I get autocompletion on the SQL statements.